dev.levz.TinyImageFinder.yml

```diff
@@ -0,0 +1,18 @@
+app-id: dev.levz.TinyImageFinder
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: image-finder
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=host
```

Comment:
line 10: this is a broad permission

Reply:
Thanks for pointing to it, "host:ro" is sufficient. Anyway, because it is a system wide image finder "host" permission is required.